api/comments/serializers.py

```diff
@@ -131,25 +126,12 @@ def get_validated_target_type(self, obj):
 
     def get_target(self, node_id, target_id):
         target = Guid.load(target_id)
-        if not target:
+        if not target or not getattr(target.referent, 'belongs_to_node'):
```

Comment:
`getattr` will raise an `AttributeError` unless you pass a default as the 3rd argument.